tools/serverpod_cli/test/generator/dart/server_code_generator/expected_files_created_test.dart

```diff
@@ -159,13 +157,8 @@ void main() {
   group(
       'Given relativeServerTestToolsPathParts is set and database enabled when generating protocol code',
       () {
-    var configWithTestToolsPath = GeneratorConfigBuilder()
-        .withName(projectName)
-        .withEnabledExperimentalFeatures(
-      [
-        ExperimentalFeature.testTools,
-      ],
-    ).withEnabledFeatures(
+    var configWithTestToolsPath =
```

Comment:
Suggestion: Rename to reflect that `testTools` is no longer enabled by a flag.

Is there something else significant with this configuration now that future readers might benefit from?

Reply:
As discussed offline this was a misunderstanding.